docs/updating_guide.md

```diff
@@ -0,0 +1,133 @@
+# Autonomous Driving Data Framework (ADDF) Version Updating Guide
+
+Autonomous Driving Data Framework follows the Industry standard `GitOps` model. The project is entirely driven by `Descriptive` prinicples in the form of asking inputs from a module developer via manifest file(s).
+
+## Steps to update ADDF
+
+### Upgrading an existing single account project deployment created with SeedFarmer v1.x should be possible with the below steps
+
+You will need to fetch the upstream ADDF repository using the below command:
+
+```bash
+git fetch upstream
+```
+
+You can merge the fetched release branch into your currently working/branch of interest using the below command:
+
+```bash
+git merge release/[MAJOR.MINOR.PATCH]
+```
+
+#### Create and activate a Virtual environment
+
+```bash
+python3 -m venv .venv && source .venv/bin/activate
+```
+
+#### Install the requirements
+
+```bash
+pip install -r ./requirements.txt
+```
+
+#### Modify the Deployment Manifest
+
+- Before you begin any changes, please check the latest format of deployment manifest [here](https://seed-farmer.readthedocs.io/en/latest/manifests.html#deployment-manifest), where you can find out information about the supported attributes
+- Add required `toolchainRegion`
+- Add required `targetAccountMappings` with at least one account
+  - Mark one account with `default: true` and one region with `default: true`
+  - add a `.env` file at the root of the addf project which has the values of global kind of variables. In the below deployment (2nd) snippets, we declare PRIMARY_ACCOUNT as an env variable, and set its value in the .env file as `PRIMARY_ACCOUNT=1234567890`
```

Comment:
see above note on providing examples of specifying a static accountId or valueFrom

Reply:
added